src/usr/local/www/system.php

```diff
@@ -467,6 +467,10 @@ function is_timezone($elt) {
 	}
 }
 
+if (($pconfig['webguicss'] == "") || (!isset($csslist[$pconfig['webguicss']]))) {
```

Comment:
I was thinking that it was always set up above even if the $config value did not exist.
You are right - isset() will still return false here.
@sbeaver-netgate has committed this already https://github.com/pfsense/pfsense/commit/c040eeda3a9710869caf276a564207866497ee67
However he does that is different to you and the PR does not get closed.
I can submit another PR to make it better...